makefiles/pseudomodules.inc.mk

```diff
@@ -1,4 +1,8 @@
 PSEUDOMODULES += at_urc
+PSEUDOMODULES += at_urc_isr
+PSEUDOMODULES += at_urc_isr_low
+PSEUDOMODULES += at_urc_isr_medium
+PSEUDOMODULES += at_urc_isr_highest
```

Comment:
```suggestion
PSEUDOMODULES += at_urc_isr_%
```

Reply:
I'd rather have the explicit list, just to know what's there. But if you insist on this I can change it.

Reply:
You are right, explicit is better than implicit.